File: c/chapter_0/1-6.c
// PTA 习题 7-9
// 求整数均值
#include <stdio.h>
int main(void){
    int a1,a2,a3,a4,sum;
    double average;
    scanf("%d%d%d%d",&a1,&a2,&a3,&a4);
    sum = a1+a2+a3+a4;
    average = sum / 4.0;
    printf("Sum = %d; Average = %.1f\n",sum,average);
    return 0;
}

File: python/chapter_0/1-6.py
# PTA 习题 7-9
# 求整数均值
a, b, c, d = map(int,input().split())
sum_value = a + b + c + d
average_value = round(sum_value / 4, 1)
print("Sum = {}; Average = {}".format(sum_value,average_value))